index.js

```diff
@@ -37,8 +64,11 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
     formResponse.innerHTML = res.join(', ');
   }
+  formFunc();
 });
 
+window.addEventListener('submit', formFunc);
```

Comment:
- [ ] Good job so far 👍🏽. Kindly note that the project requirement expects that after filling out the form, and not submitting it, the entered values should be saved in the browser and to achieve that, you have to save the data on input(Save the data to local storage as you type).

you are saving yours to local storage after the form is submitted which is not what the project requirement specifies.

e.g the `formElements.addEventListener('input', formFunc);`

Reply:
Fixed